package.json

```diff
@@ -44,6 +44,10 @@
     "mochawesome": "2.3.1",
     "rewire": "3.0.2",
     "should": "13.1.3",
-    "sinon": "^6.3.4"
+    "sinon": "^6.3.4",
+    "prom-client": "^11.2.1"
+  },
+  "peerDependencies": {
+    "prom-client": "11.x"
   }
```

Comment:
it's peer because I'm "forcing" the user to have the "prom-client" package running in his project (because i'm only registering my metrics to prom and it's his responsibility to init it and to expose it) it appears twice because I need it for the tests.